zds/member/views.py

```diff
@@ -581,8 +577,8 @@ def settings_mini_profile(request, user_name):
             profile.avatar_url = form.data['avatar_url']
             profile.sign = form.data['sign']
 
-            # Save the profile and redirect the user to the configuration space
-            # with message indicate the state of the operation
+            # Save the profile and redirect the user to the configuration
+            # space with message indicate the state of the operation.
```

Comment:
`Save profile and redirect user to the settings page with a message indicating the operation state.`